deebot_client/commands/json/map.py

```diff
@@ -290,6 +326,42 @@ def _handle_response(
         return result
 
 
+class GetMajorMap(JsonCommandWithMessageHandling, MessageBodyDataDict):
```

Comment:
Could you let me know why you moved this class? They are sorted alphabetically

Reply:
I have move it because of logical connection between the function as i tried to understand how the map draw is working. but in #373 it was moved back to sorted order.

Reply:
Moved it back to have a smaller changeset in this PR.
I think soon or later we need to create a module out of map